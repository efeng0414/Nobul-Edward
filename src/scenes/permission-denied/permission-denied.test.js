import React from "react";
import renderer from "react-test-renderer";
import { IntlProvider } from "react-intl";
import PermissionDenied from "./container";

describe("Permission Denied", () => {
  const containerProps = {
    intl: global.getIntl()
  };

  const containerJSX = (
    <IntlProvider {...containerProps}>
      <PermissionDenied />
    </IntlProvider>
  );

  describe("Container Snapshot", () => {
    const snapshotJSX = (
      <IntlProvider {...global.intl}>{containerJSX}</IntlProvider>
    );

    const containerSnapshot = renderer.create(snapshotJSX).toJSON();
    it("should match stored snapshot", () => {
      expect(containerSnapshot).toMatchSnapshot();
    });
  });
});
