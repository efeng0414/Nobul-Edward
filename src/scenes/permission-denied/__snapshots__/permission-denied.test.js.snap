// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Permission Denied Container Snapshot should match stored snapshot 1`] = `
<div
  className="permission-denied"
>
  <h1>
    Oops, you're not authorized to be here!
  </h1>
  <p>
    Some pages are restricted for consumers and agents, please double check you are logged in with the correct credentials.
  </p>
</div>
`;
